WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -52,6 +50,7 @@ public partial class SendViewModel : RoutableViewModel
 		[AutoNotify] private ObservableCollection<string> _labels;
 		[AutoNotify] private bool _isPayJoin;
 		[AutoNotify] private string? _payJoinEndPoint;
+		[AutoNotify] private bool _isQrButtonVisible;
```

Comment:
I assume the value of the property cannot change during runtime so no need to be `AutoNotify`.
Can you make it to a simple prop? 
```
public bool IsQrButtonVisible { get; }
```

Although a super tiny modification it makes it a bit understandable. 
A dev who is not familiar with this code could think that the visibility of the button can change because there is the AutoNotify tag.

Reply:
Sure, done.